internal/cli/kraft/compose/create/create.go

```diff
@@ -466,8 +466,20 @@ func createService(ctx context.Context, project *compose.Project, service types.
 	log.G(ctx).Infof("creating service %s...", service.Name)
 
 	networks := []string{}
+	if len(service.DNS) > 2 {
+		log.G(ctx).Warnf("service %s has more than 2 DNS servers, only the first 2 will be used", service.Name)
+	}
+
+	dns0 := ""
+	dns1 := ""
+	if len(service.DNS) > 0 {
+		dns0 = service.DNS[0]
+	}
+	if len(service.DNS) > 1 {
+		dns1 = service.DNS[1]
+	}
 	for name, network := range service.Networks {
-		networks = append(networks, fmt.Sprintf("%s:%s", project.Networks[name].Name, network.Ipv4Address))
+		networks = append(networks, fmt.Sprintf("%s:%s::%s:%s:%s:%s", project.Networks[name].Name, network.Ipv4Address, dns0, dns1, service.Hostname, service.DomainName))
```

Comment:
This syntax scares me.

Probably it should be possible to pass these details in an structured way to the underlying run command?

Reply:
I pushed a different version that uses the ``NetdevIp`` parameter. Let me know if this is better